FCCee/CAD_beampipe/compact/Beampipe_CADimport.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+
+
+  <info name="FCCee"
+        title="FCCee Beam pipe from CAD drawings"
+        author="aciarma"
+        url="no"
+        status="development"
+        version="1.0">
+    <comment>A beampipe for FCCee coming from F. Fransesini CAD drawings. 
+             CAD model -2.5m/+2.5m from the IP, after simple DD4hep tubes. </comment>
+  </info>
+
+
+  <display>
+    <vis name="BeamPipeVis" alpha="0.0" r="0.0" g="1.0" b="0.0" showDaughters="true" visible="false"/>
+    <vis name="GoldCoatingVis" alpha="0.0" r="0.0" g="1.0" b="1.0" showDaughters="true" visible="true"/>
+    <vis name="TubeVis"  alpha="1.0" r="1.0" g="0.7"  b="0.5"   showDaughters="true"  visible="true"/>
+    <vis name="VacVis"   alpha="1.0" r="1.0" g="1.0"  b="1.0"   showDaughters="true"  visible="false"/>
+  </display>
+
+
+  <detectors>
+
+    <detector id="1" name="Beampipe_STL" type="DD4hep_TestShape_Creator">
```

Comment:
Is it possible to remove the 'id' there or is it crashing? It may conflict with id defined in other detectors when importing this xml...

Reply:
done for all